nats-streaming-server.go

```diff
@@ -127,7 +127,7 @@ func parseFlags() (*stand.Options, *natsd.Options) {
 	flag.StringVar(&stanOpts.ClientCert, "tls_client_cert", "", "Path to a client certificate file")
 	flag.StringVar(&stanOpts.ClientKey, "tls_client_key", "", "Path to a client key file")
 	flag.StringVar(&stanOpts.ClientCA, "tls_client_cacert", "", "Path to a client CA file")
-	flag.BoolVar(&stanOpts.EmbedNATS, "embed", true, "Embed the NATS Server")
+	flag.BoolVar(&stanOpts.EmbedNATS, "embed_nats", stand.DefaultEmbedNATS, "Embed the NATS Server")
```

Comment:
There was only one (it was renamed, this is why you see 2 lines ;-) ). But, no problem, I can indeed replace this with `-server` and the user would provide the url to the external NATS Server. Would the user be responsible from providing the user/pwd in that URL or not? The reason I ask is that currently we build the URL based on the provided NATS options.